src/style.css

```
@@ -0,0 +1,180 @@
+:root {
+    --black: #1b1b1b;
+    --blue: #3aa3e9;
+    --gray: #bcc2c6;
+    --light-gray: #ddd;
+    --yellow: #f7df1e;
+    --red: #eb4d4b;
+    --green: #27ae60;
+}
+
+body {
+    color: var(--black);
+    margin: 0;
+    font-family: 'Poppins', sans-serif;
+    box-sizing: border-box;
+    height: 100vh;
+}
+
+body * {
+    box-sizing: inherit;
+    outline: none;
+}
+
+button {
+    font-family: inherit;
+}
+
+a,
+a:visited {
+    color: var(--black);
+    text-decoration: none;
+    border-bottom: .1em solid var(--gray);
+}
+
+button {
+    border: 1px solid var(--black);
+    background: none;
+    cursor: pointer;
+}
+
+button:hover {
+    background: var(--black);
+    color: #fff;
```

Uso de variables. Antes de aplicar este cambio, es necesario agregarla
```suggestion
    color: var(--white);
```